### PR TITLE
Feature/task 129  and 127- custom Logger, Configuration class on utils

### PR DESCRIPTION
**Pull request by strymsg**
Done basic configuration for flask app and loading of configuration files in task-scheduler/configs/
```
task_scheduler/configs
├── config.dev.json
├── config.dev.sample.json
├── config.prod.json
├── config.prod.sample.json
└── config.py
```
(Only need to change config.dev.json)

To be able to run flask with **pycharm** need:

1. Edit run configuration
2. Add flask config and set the following
3. script path: to venv/bin/flask
4. Parameters: run
5. Envirnoment variables: FLASK_APP=task_scheduler; FLASK_ENV=development; FLASK_DEBUG=1
6. Working directory: root directory of task_scheduler app, for instance: `/home/ubuntu/prog102-AT-projects/task-scheduler-at`

Example of import and use loggin:
```python
from task_scheduler.utils.logger import CustomLogger
# ...
self.logger = CustomLogger(__name__)
self.logger.info(f"some message")
```

### CODE REVIEW

File: task_scheduler/utils/logger.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import logging
+
+
+class CustomLogger(logging.Logger):
+    # Custom Logger should be created when having an app taht has many modules
+    # or your app uses part time modules
```
Comment (favipcj) [inline task_scheduler/utils/logger.py]: fyi.  this was only for an example. Please use docstrings to document the goal of this class

File: task_scheduler/__init__.py [hunk 2]
```diff
@@ -3,3 +3,34 @@
...
+    app = Flask(__name__, instance_relative_config=True)
+    print("task scheduler...")
+    config_obj = Configuration()
+
+    logger = config_logger(config_obj.configuration)
+    #logger.info(f'Configurations loaded {config_obj.configuration}')
+    print("logger...", logger)
+    logger.info("logging info test")
+    logger.error("logging Error test")
+    logger.debug("logging debug test")
```
Comment (favipcj) [inline task_scheduler/__init__.py]: before merging this PR, please be sure to clean up these tests

File: task_scheduler/configs/config.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+class Configuration:
+    '''Loads the configuration json file (depending on FLASK_ENV variable)
+     and returns a dict. If loading fails prints error and returns enmpty dict
+    '''
+    def __init__(self):
+        self.configuration = {}
+        filename = ''
+        if os.environ.get('FLASK_ENV') == 'development':
+            filename = 'task_scheduler/configs/config.dev.json'
```
Comment (favipcj) [inline task_scheduler/configs/config.py]: maybe here you can use `path` to get the absolute path, somthign like the following
`absulte_path= path.join(path.abspath(path.dirname(__file__)), 'config.dev.json') `

File: task_scheduler/configs/config.dev.sample.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (favipcj) [inline task_scheduler/configs/config.dev.sample.json:1]: As you are adding files that are not python files, please be sure to add these path files on your MANIFEST.in file

File: task_scheduler/configs/config.py (hunk 3 above)
Comment (strymsg) [reply task_scheduler/configs/config.py]: Thanks the suggestions have been applied.

**Pull request merged**